po/es.po

```diff
@@ -6,7 +6,7 @@ msgstr ""
 "PO-Revision-Date: 2021-09-14 15:25\n"
 "Last-Translator: \n"
 "Language-Team: Spanish\n"
-"Language: es_ES\n"
+"Language: es\n"
```

Comment:
This one is also autogenerated and would require manual change of configuration at Crowdin first. Considering you changed back the other one, I guess this one should also remain as `es_ES`, unless there is a reason you want to change this?

Reply:
This is still changed, and you resolved conversation without answering any of my questions.